doc/usermanual/darkroom/concepts/raster_masks.xml

```diff
@@ -13,50 +13,54 @@
   </indexterm>
 
   <para>
```

Comment:
Personally, I think the original wording here is the better option. Sure, it's a little more verbose, but I think the subject matter warrants the deeper explanation. Just my 2 cents.

Reply:
Thanks for your feedback. My main doubts about the original lines 16-19 are as follows:
1. I am not sure what it means for a mask to be active.
My understanding is that a a specific mask option (uniform, drawn, parametric, or drawn and parametric) needs to be enabled in the context of a given module.
Does enabling a mask option make a mask active? I'm not sure.
- Enabling a drawn mask without drawing a single shape already makes it available as a raster mask down the pixelpipe. Can an empty drawn mask be considered active?
- The same question applies to a parametric mask in default state as it is also made available down the pixelpipe.
-  Disabling a module with a mask still leaves the raster mask available down the pixelpipe (at least in the drop-down). Can a mask in disabled module be considered active?
2. I do not understand a reference to blend functions in the context of a parametric mask.
Blending operations/modes (normal, lighten, darken, etc.) are available for any mask type. Parametric mask controls help in defining opacity of what the manual calls a blend mask (again possible clash with blend modes) via selection curves. Maybe if we start referring to selection curves of parametric masks, we can prevent this clash of the terms at least in the discussed paragraph. Or maybe my perception is wrong and there is no conflict here.
3. I am not sure if the notion of a final mask is used in other parts of the manual.
If not, it seems to be needed only for a specific use case which kind of pollutes the domain language.
4. I have a doubt about the phrasing "..shapes.. and blending curves.. assemble the final mask".
I would say they contribute to final mask, as assembly implies a union, while a combined drawn and parametric mask allow easily excluding areas which distinct masks would select.
5. I am concerned that we are mentioning only drawn or parametric masks.
Uniform masks on modules also appear in raster mask drop-down. Of course, their use case is rather primitive (just like the ones of default drawn or parametric mask). BTW, this also introduces an issue to the phrasing I am suggesting, so it would need to be fixed.
Drawn and parametric masks are implied by mentioning drawn masks and parametric ones, but I wonder if they should be mentioned explicitly.

Any constructive comments are welcome.